src/ui/QuerySuggestPreview/QuerySuggestPreview.ts

```diff
@@ -22,6 +22,20 @@ export interface IQuerySuggestPreview {
   executeQueryDelay?: number;
 }
 
+/**
+ * This component adds previews of the top query results matching the currently focused query suggestion in the search box.
+ *
+ * As such, this component only works when an [`Omnibox`]{@link Omnibox} is present in the search interface.
+ *
+ * Moreover, this component can use a [result template](https://docs.coveo.com/en/413/) in its markup configuration to render previews with.
+ *
+ * **Example**
+ * ```
+ *   <div class="CoveoQuerySuggestPreview"></div>
+ * ```
+ *
+ * See [Providing Query Suggestion Result Previews](https://docs.coveo.com/en/340/#providing-query-suggestion-result-previews).
```

Comment:
`#providing-query-suggestion-result-previews` doesn't exist in https://docs.coveo.com/en/340, should I remove this link, change it or keep it?

Reply:
This will be a question for @fbeaudoincoveo . I'm assuming that it was hidden from the public documentation since the component was delayed and not made available when the doc was written.

There will be some coordination we need to do here I guess...

Reply:
The external documentation actually hasn't been written yet; I was waiting for the feature to be closer to its final state before doing it.

To coordinate things, I'll simply publish the doc at the same time I publish the public release notes for the package that contains this new feature. Assuming the reference documentation is updated at the same time as the release happens, there should only be a short gap (like a couple of hours) during which the link won't work, which should be no big deal.